pilot/docker/prepare_proxy.sh

```diff
@@ -28,6 +30,9 @@ while getopts ":p:u:e:i:h" opt; do
     i)
       IP_RANGES_INCLUDE=${OPTARG}
       ;;
+    g)
+      REDIRECT_INBOUND_TRAFFIC="no" # For gateways, do not redirect inbound traffic
```

Comment:
Please remove gateway, as it implies that we need IPtables for gateways.. What you need here is a customization for the use case you point to (trap outbound but not inbound). That could apply to any normal pod as well.